sklearn/tests/test_multiclass.py

```diff
@@ -824,6 +824,61 @@ def test_pairwise_indices():
         )
 
 
+def test_pairwise_n_features_in():
+    """Check the n_features_in_ attributes of the meta and base estimators
+
+    When the training data is a regular design matrix, everything is intuitive.
+    However, when the training data is a precomputed kernel matrix, the
+    multiclass strategy can resample the kernel matrix of the underlying base
+    estimator both row-wise and column-wise and this has a non-trivial impact
+    on the expected value for the n_features_in_ of both the meta and the base
+    estimators.
+    """
+    X, y = iris.data, iris.target
+
+    # Fitting directly on the design matrix:
+    assert X.shape == (150, 4)
+
+    clf_notprecomputed = svm.SVC(kernel="linear").fit(X, y)
+    assert clf_notprecomputed.n_features_in_ == 4
+
+    ovr_notprecomputed = OneVsRestClassifier(clf_notprecomputed).fit(X, y)
+    assert ovr_notprecomputed.n_features_in_ == 4
+    for est in ovr_notprecomputed.estimators_:
+        assert est.n_features_in_ == 4
+
+    # Fitting the same models with a precomputed kernel:
+    K = X @ X.T
+    assert K.shape == (150, 150)
```

Comment:
Should you move these lines below the next assert, closer to where it is used?

Reply:
I was going to say the same thing :)